pkg/controller/reconciler/identitymanager_reconciler.go

```diff
@@ -56,6 +56,8 @@ func (z *identityManagerReconciler) Reconcile(node *resourcetree.ResourceNode) (
 			HostedZoneID: resourceState.HostedZoneID,
 		})
 		if err != nil {
+			result.Requeue = errors.IsKind(err, errors.Timeout)
```

Comment:
This can create an eternal loop if there are endless of timeout errors.  Perhaps it can happen if we for instance have too low timeout, but there might be other unknown reason that leads to a timeout error.

Just mentioning, if you think it is safe enough, leave it. If not, we need a maximum retries thingy here.

Reply:
The maximum retries thingy happens [here](https://github.com/oslokommune/okctl/blob/f4fa56da142ac97ff9f77d1b2c397fe8082e39be/pkg/controller/synchronize.go#L71). This is just a signal indicating what the reconciler wants to do.

Reply:
Ooh right, totally forgot that. Good then :+1: